src/vivarium_gates_mncnh/components/intervention.py

```diff
@@ -7,68 +7,89 @@
 from vivarium.framework.engine import Builder
 
 from vivarium_gates_mncnh.constants import data_values
-from vivarium_gates_mncnh.constants.data_keys import NO_CPAP_RISK
 from vivarium_gates_mncnh.constants.data_values import COLUMNS, PIPELINES
-from vivarium_gates_mncnh.utilities import get_location
 
 
-class NoCPAPRisk(Component):
-    """Component for CPAP intervention. This is essentially a risk effect."""
+class NeonatalNoInterventionRisk(Component):
```

Comment:
This is just a refactor since CPAP and antibiotics are the same implementation so the component can just be created in the config file